app/packs/src/apps/mydb/inbox/InboxModal.js

```diff
@@ -441,7 +443,9 @@ export default class InboxModal extends React.Component {
                     </div>
 
                   </div>
-                  <div className={`tree-wrapper${inboxVisible ? 'd-none' : ''}`}>
+                  <div
+                    className={`tree-wrapper${inboxVisible ? 'd-none' : ''}`}
```

Comment:
If this cannot be converted to `{inboxVisible && <div…}`, there is a space missing between the class names. 😊